packages/vite/src/node/plugins/importAnalysisBuild.ts

```diff
@@ -515,12 +515,13 @@ export function buildImportAnalysisPlugin(config: ResolvedConfig): Plugin {
                   const chunk = bundle[filename] as OutputChunk | undefined
                   if (chunk) {
                     deps.add(chunk.fileName)
-                    chunk.imports.forEach(addDeps)
+                    if (chunk.imports) chunk.imports.forEach(addDeps)
```

Comment:
Is the correct fix here to check if `chunk.type` === `'chunk'` and not `'asset'` instead? The types for `OutputChunk` show that `imports` should exist. We could add that check around this block here.

Reply:
It seems like this solves the issue as well

Reply:
Implemented in https://github.com/vitejs/vite/pull/15469/commits/eea60838090ddae6b634c693b247a81826546e5b